addons/csw/CfgWeapons.hpp

```diff
@@ -179,6 +179,20 @@ class CfgWeapons {
         modes[] = {};
         picture = QPATHTOF(UI\StaticHGMG_Icon.paa);
     };
+    class GVAR(staticM2ShieldCarry): GVAR(staticHMGCarry) {
+        class ADDON: ADDON {
+            class assembleTo {
+                GVAR(m3Tripod) = "B_HMG_02_high_F";
+                GVAR(m3TripodLow) = "B_HMG_02_F";
+            };
+        };
+        class WeaponSlotsInfo: WeaponSlotsInfo {
+            mass = 1000;
```

Comment:
I think this means the mass is defined as 100lbs. In real life the mass is 84lbs according to wIkipedia

Reply:
I was thinking the shield should add some heft

Reply:
yeah fair actually